test/unit/src/server/handlers/t_login.js

```diff
@@ -1,9 +1,15 @@
 import expect from 'expect';
```

Comment:
Changes in the `src` directory are fairly clean. Not so with the `test` directory, which screws around with config in order to test stuff.

The changes in this directory are mostly the replacement of `conf.stores` with proxyquire, and tidying up test files after the additional boilerplate had been added.